startup/11-shutters.py

```diff
@@ -52,6 +52,8 @@ def shutter_cb(value, timestamp, **kwargs):
         return st
 
     def stop(self, success):
+        # MR: commenting out on 10/30/2018 for manual control of the shutters (see #10).
+        '''
```

Comment:
Can you put in leading `#` instead of string comments?  This makes the diffs very hard to read....

Reply:
Sure, I will. Thanks @tacaswell!